### ISSUE TITLE
Maybe we should tidy up duplicate icons and possibly further things

### ISSUE THREAD

**Issue by Photon89** [shutter-project/shutter#381]
As noted in https://bugzilla.redhat.com/show_bug.cgi?id=1982210#c2 there seem to be several icons which are placed in different places. Probably the duplicates should be substituted by symlinks.

There are also further proposals, for example, an invalid locale.

@martinkg If you have any input on what we should do upstream to make Fedora packaging easier, please comment!

**Comment by martinkg** [comment]
also reported as a bug 
[wrong icon size](https://github.com/shutter-project/shutter/issues/384)

**Comment by Photon89** [comment]
@martinkg 

In https://github.com/shutter-project/shutter/pull/383 I fixed the 24px icon size and substituted the files which have been reported as duplicate by rpmlint with symlinks. 

It also complained about a locale, I looked it up and it seems to be this one: https://translations.launchpad.net/shutter/trunk/+lang/ber Not really complete at all, but still seems to be a valid locale to me.

Also it complained about a missing man page, but we do have a man page, any ideas what is wrong about it?
